src/main/java/com/softserve/paymentservice/dto/CardDto.java

```diff
@@ -0,0 +1,18 @@
+package com.softserve.paymentservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CardDto {
+    private UUID userUUID;
+    private String cardNumber;
+    private String expMonth;
+    private String expYear;
```

Comment:
I'd rather say, that you should save date in date variables - Instant for example;

Reply:
Instant has time, so it shouldn't be here. As well as LocalDate as it contains days.

Which format should month have? I think we definitely need to ensure month, cardNumber and year are always valid. It can be done using types and/or bean validation